encrypted-media/clearkey-mp4-playback-temporary-encrypted-clear-sources.html

```diff
@@ -2,7 +2,7 @@
 <html>
   <head>
     <meta charset=utf-8>
-    <title>Encrypted Media Extensions: Successful Playback, Temporary session with Clear Key, mp4, Encrypted and Clear sources</title>
+    <title>Encrypted Media Extensions: Successful Playback, alternate, separate, Encrypted and Clear sources, Temporary, mp4, Clear Key</title>
```

Comment:
Is "alternate, separate, Encrypted and Clear sources" one thing? These blend in with other comma-separated items.